python/lsst/ts/fiberspectrograph/dataManager.py

```diff
@@ -105,19 +106,27 @@ def make_fits_header(self, data):
         # TODO: it would be good to include the dataclass docstrings
         # as comments on each of these, but pydoc can't see them.
         hdr["FORMAT_V"] = FORMAT_VERSION
-        hdr["OBSERVAT"] = "Vera C. Rubin Observatory"
-        hdr["INSTRUME"] = self.instrument
-        hdr["ORIGIN"] = self.origin
-        hdr["LOCATN"] = (None, "Location of Instrument")
+        hdr["ORIGIN"] = "Vera C. Rubin Observatory"
+        hdr["INSTRUME"] = (self.instrument, "Type of Instrument.")
+        hdr["SERIAL"] = (self.serial, "Serial Number of Spectrograph.")
+        hdr["CSCNAME"] = (self.origin, "Name of the CSC that produced this data.")
+        hdr["LOCATN"] = (None, "Location of Instrument.")
         hdr["DETSIZE"] = data.n_pixels
         hdr["DATE-BEG"] = astropy.time.Time(data.date_begin).tai.fits
         hdr["DATE-END"] = astropy.time.Time(data.date_end).tai.fits
-        hdr["EXPTIME"] = data.duration
+        hdr["DAYOBS"] = astropy.time.Time(data.date_begin).tai.strftime("%Y%m%d")
+        hdr["EXPTIME"] = (data.duration, "Duration of scan.")
         hdr["TIMESYS"] = "TAI"
-        hdr["IMGTYPE"] = data.type
+        hdr["IMGTYPE"] = "spectrum"  # Temporary
```

Comment:
Is there a ticket for determining all the actual options for IMGTYPE? Adding the ticket number to the comment here would be useful.

Reply:
Just opened a ticket for this. [DM-38311](https://jira.lsstcorp.org/browse/DM-38311)